build.gradle

```diff
@@ -43,7 +43,7 @@ repositories {
 // 'api' configuration allows to expose dependencies with 'compile' scope in pom
 // 'implementation' configuration allows to expose dependencies with 'runtime' scope in pom
 dependencies {
-    implementation platform("org.springframework.boot:spring-boot-dependencies:2.6.2")
+    implementation platform("org.springframework.boot:spring-boot-dependencies:2.6.14")
```

Comment:
`api 'org.apache.commons:commons-lang3'`

Reply:
Thanks :+1: You can replace double quotes too*

Reply:
Sure. I removed all the double quotes that were useless. Please see https://github.com/iExecBlockchainComputing/iexec-common/pull/375/commits/9cbea402743d87cc205bd4be32aab91c775568da.

Reply:
Better thanks!